<!DOCTYPE html>
<html lang="en-us">

<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 4.3.1">

  

  
  
  
  
  
    
    
    
  
  

  <meta name="author" content="Liliana M. Dávalos">

  
  
  
    
  
  <meta name="description" content="Online home of the Davalos Lab">

  
  <link rel="alternate" hreflang="en-us" href="/tags/bayesian-phylogenetics/">

  


  

  
  
  
  <meta name="theme-color" content="#3f51b5">
  

  
  
  
  
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.0/css/all.css" integrity="sha384-aOkxzJ5uQz7WBObEZcHvV5JvRW3TUc2rNPA7pe3AwnsUohiw1Vj2Rgx2KSOkF5+h" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">

    
    
    
      
    
    
      
      
        
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/github.min.css" crossorigin="anonymous" title="hl-light">
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/dracula.min.css" crossorigin="anonymous" title="hl-dark" disabled>
        
      
    

    

    

  

  
  
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:400,700|Roboto:400,400italic,700|Roboto+Mono&display=swap">
  

  
  
  
  <link rel="stylesheet" href="/css/academic.min.05203e7e5f39578bc40dee07dafc7ac6.css">

  

  
  
  

  
  <link rel="alternate" href="/tags/bayesian-phylogenetics/index.xml" type="application/rss+xml" title="Dávalos Lab">
  

  <link rel="manifest" href="/site.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="/tags/bayesian-phylogenetics/">

  
  
  
  
    
  
  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="twitter:site" content="@LabDavalos">
  <meta property="twitter:creator" content="@LabDavalos">
  
  <meta property="og:site_name" content="Dávalos Lab">
  <meta property="og:url" content="/tags/bayesian-phylogenetics/">
  <meta property="og:title" content="Bayesian phylogenetics | Dávalos Lab">
  <meta property="og:description" content="Online home of the Davalos Lab"><meta property="og:image" content="/img/DSC00813">
  <meta property="twitter:image" content="/img/DSC00813"><meta property="og:locale" content="en-us">
  
  <meta property="og:updated_time" content="2018-01-24T00:00:00&#43;00:00">
  

  


  





  <title>Bayesian phylogenetics | Dávalos Lab</title>

</head>

<body id="top" data-spy="scroll" data-offset="70" data-target="#TableOfContents" >

  <aside class="search-results" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search">
        
      </div>

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>


  
<nav class="navbar navbar-light fixed-top navbar-expand-lg py-0 compensate-for-scrollbar" id="navbar-main">
  <div class="container">

    
      <a class="navbar-brand" href="/">Dávalos Lab</a>
      
      <button type="button" class="navbar-toggler" data-toggle="collapse"
              data-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
        <span><i class="fas fa-bars"></i></span>
      </button>
      

    
    <div class="collapse navbar-collapse" id="navbar">

      
      
      <ul class="navbar-nav mr-auto">
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#about"><span>Home</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#people"><span>People</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#projects"><span>Projects</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#publications"><span>Publications</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/courses/"><span>Courses</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#contact"><span>Contact</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#join_lab"><span>Join</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#blog"><span>Blog</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/files/Davalos_CV_current.pdf"><span>CV</span></a>
        </li>

        
        

      
      </ul>
      <ul class="navbar-nav ml-auto">
      

        

        
        <li class="nav-item">
          <a class="nav-link js-search" href="#"><i class="fas fa-search" aria-hidden="true"></i></a>
        </li>
        

        

        
        <li class="nav-item">
          <a class="nav-link js-dark-toggle" href="#"><i class="fas fa-moon" aria-hidden="true"></i></a>
        </li>
        

      </ul>

    </div>
  </div>
</nav>


  












  

  
  
  
    
  
<div class="universal-wrapper pt-3">
  <h1 itemprop="name">Bayesian phylogenetics</h1>

  

  
</div>



<div class="universal-wrapper">
  

  
  
  <div>
    <h2><a href="/publication/journal_out_of_the_antilles/">Out of the Antilles: Fossil Phylogenies Support Reverse Colonization of Bats to South America</a></h2>
    <div class="article-style">
      
      Previous phylogenies of extant short‐faced bats (Chiroptera Stenodermatina) supported either two colonization events from the mainland to the Antilles, or reverse colonization, but lacked both fossil data and statistical modelling of biogeography. …
      
    </div>
  </div>
  
  <div>
    <h2><a href="/publication/journal_bats_chiroptera_noctilionoidea/">Bats (Chiroptera: Noctilionoidea) Challenge a Recent Origin of Extant Neotropical Diversity</a></h2>
    <div class="article-style">
      
      The mechanisms underlying the high extant biodiversity in the Neotropics have been controversial since the 19th century. Support for the influence of period-specific changes on diversification often rests on detecting more speciation events during a …
      
    </div>
  </div>
  
  <div>
    <h2><a href="/publication/journal_integrating_incomplete_fossils/">Integrating Incomplete Fossils by Isolating Conflicting Signal in Saturated and Non-Independent Morphological Characters</a></h2>
    <div class="article-style">
      
      Morphological characters are indispensable in phylogenetic analyses for understanding the pattern, process, and tempo of evolution. If characters are independent and free of systematic errors, then combining as many different kinds of characters as …
      
    </div>
  </div>
  
  <div>
    <h2><a href="/publication/journal_dynamic_recruitment_of_amino_acid_transporters/">Dynamic recruitment of amino acid transporters to the insect/symbiont interface</a></h2>
    <div class="article-style">
      
      Symbiosis is well known to influence bacterial symbiont genome evolution and has recently been shown to shape eukaryotic host genomes. Intriguing patterns of host genome evolution, including remarkable numbers of gene duplications, have been observed …
      
    </div>
  </div>
  
  <div>
    <h2><a href="/publication/journal_understanding_phylogenetic_incongruence/">Understanding phylogenetic incongruence: lessons from phyllostomid bats</a></h2>
    <div class="article-style">
      
      All characters and trait systems in an organism share a common evolutionary history that can be estimated using phylogenetic methods. However, differential rates of change and the evolutionary mechanisms driving those rates result in pervasive …
      
    </div>
  </div>
  
  <div>
    <h2><a href="/publication/journal_morphological_innovation_diversification/">Morphological innovation, diversification and invasion of a new adaptive zone</a></h2>
    <div class="article-style">
      
      How ecological opportunity relates to diversification is a central question in evolutionary biology. However, there are few empirical examples of how ecological opportunity and morphological innovation open new adaptive zones, and promote …
      
    </div>
  </div>
  
  <div>
    <h2><a href="/publication/journal_accounting_for_molecular_stochasticity/">Accounting for molecular stochasticity in systematic revisions: Species limits and phylogeny of Paroaria</a></h2>
    <div class="article-style">
      
      Different frameworks have been proposed for using molecular data in systematic revisions, but there is ongoing debate on their applicability, merits and shortcomings. In this paper we examine the fit between morphological and molecular data in the systematic revision of Paroaria, a group of conspicuous song- birds endemic to South America. We delimited species based on examination of 600 specimens, and developed distance-gap, and distance- and character-based coalescent simulations to test species limits with molecular data. The morphological and molecular data collected were then analyzed using parsi- mony, maximum likelihood, and Bayesian phylogenetics. The simulations were better at evaluating the new species limits than using genetic distances. Species diversity within Paroaria had been underesti- mated by 60%, and the revised genus comprises eight species. Phylogenetic analyses consistently recov- ered a congruent topology for the most recently derived species in the genus, but the most basal divergences were not resolved with these data. The systematic and phylogenetic hypotheses developed here are relevant to both setting conservation priorities and understanding the biogeography of South America.
      
    </div>
  </div>
  
  <div>
    <h2><a href="/publication/journal_short_faced_bats/">Short‐faced bats (Phyllostomidae: Stenodermatina): a Caribbean radiation of strict frugivores</a></h2>
    <div class="article-style">
      
      Aim To test the hypothesis that Caribbean Short‐faced bats descended from a single recent ancestor that originated in the continental Neotropics (Mexico, Central America and/or South America). Location The Neotropics, including the West Indies. Methods New mitochondrial cytochrome b and nuclear Rag2 sequences were combined with published molecular data to estimate phylogenetic relationships and sequence divergence among Short‐faced bats. The resulting phylogenies were compared with those compatible with the single‐origin hypothesis using two model‐based statistical tests. Confidence limits on sequence divergence were estimated using a parametric bootstrap. Results All molecular phylogenies revealed two independent Caribbean lineages and showed that continental Short‐faced bats share a recent common ancestor. Morphology‐based trees compatible with the single‐origin hypothesis were significantly worse at explaining the molecular data than any molecular phylogeny. Main conclusions The ancestor of all Short‐faced bats reached the Antilles in the Miocene, too recently to have used a proposed Oligocene land bridge, and well before the Pleistocene glaciations that are thought to have facilitated dispersal for many bats. After a long period of isolation, Short‐faced bats diversified quickly on the Caribbean islands. A single Short‐faced lineage then reached the continent and subsequently expanded its range and diversified into the four extant genera. Among bats, independent lineages of aerial insectivores and nectarivores have also recolonized the continent after evolving in the West Indies. The evidence for an insular origin of the short‐faced frugivorous radiation completes a dynamic model of Caribbean biogeography that encompasses an entire biological community.
      
    </div>
  </div>
  
  <div>
    <h2><a href="/publication/journal_geography_diversification_mormoopids/">The geography of diversification in the mormoopids (Chiroptera: Mormoopidae)</a></h2>
    <div class="article-style">
      
      The traditional explanation of the distribution of the Mormoopidae is that this family originated in southern Central America or northern South America, later expanding its range north to Mexico and the West Indies, and differentiating into eight species. An alternative fossil-based hypothesis argues that the family originated in the northern Neotropics, reached the Caribbean early in its history, and dispersed to South America after the completion of the Isthmus of Panama. The present study analyses new and previously published sequence data from the mitochondrial 12S, tRNAval, 16S, and cytochrome b, and the nuclear Rag2, to evaluate species boundaries and infer relationships among extant taxa. Fixed differences in cytochrome b often coincide with published morphological characters and show that the family contains at least 13 species. Two additional, morphologically indistinct, lineages are restricted to Suriname and French Guiana. Phylogeny-based inferences of ancestral area are equivocal on the geographical origin of mormoopids, in part because several internal nodes are not resolved with the available data. Divergences between Middle American and Antillean populations are greater than those between Mexico/Central America and South America. This suggests that mormoopids diversified in northern Neotropics before entering South America. A northern neotropical origin for mormoopids is congruent with both the Tertiary fossil record and recent phylogenetic hypotheses for the sister family to the Mormoopidae, the Phyllostomidae.
      
    </div>
  </div>
  
  <div>
    <h2><a href="/publication/journal_molecular_phylogeny_funnel-eared_bats/">Molecular phylogeny of funnel-eared bats (Chiroptera: Natalidae), with notes on biogeography and conservation</a></h2>
    <div class="article-style">
      
      Two assumptions have framed previous systematic and biogeographic studies of the family Natalidae: that it comprises a few widespread species, and that extant lineages originated in Mexico and/or Central America. This study analyzes new sequence data from the mitochondrial cytochrome b and the nuclear Rag2, to clarify species boundaries and infer relationships among extant taxa. Fixed differences in cytochrome b coincide with published morphological characters, and show that the family includes at least eight species. One newly recognized species is known to live from a single locality in Jamaica, suggesting immediate conservation measures and underscoring the urgency of taxonomic revision. Among the three genera, Chilonatalus and Natalus form a clade, to the exclusion of Nyctiellus. This phylogeny and the geographic distribution of natalids, both extant and extinct, are hardly compatible with a Middle American origin for the group. Instead, extant natalids appear to have originated in the West Indies. The threat of Caribbean hurricanes early in their evolutionary history might account for the specialized cave roosting that characterizes all natalids, even continental species.
      
    </div>
  </div>
  

  
<nav>
  <ul class="pagination justify-content-center">
    
    
    <li class="page-item"><a class="page-link" href="/tags/bayesian-phylogenetics/page/2/">&raquo;</a></li>
    
  </ul>
</nav>


</div>

      

    
    

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js" integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js" integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>

      

      
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/highlight.min.js" integrity="sha256-aYTdUrn6Ow1DDgh5JTc3aDGnnju48y/1c8s1dgkYPQ8=" crossorigin="anonymous"></script>
        
      

      
      
    

    
    

    
    
    

    
    
    <script>hljs.initHighlightingOnLoad();</script>
    

    
    
    <script>
      const search_index_filename = "/index.json";
      const i18n = {
        'placeholder': "Search...",
        'results': "results found",
        'no_results': "No results found"
      };
      const content_type = {
        'post': "Posts",
        'project': "Projects",
        'publication' : "Publications",
        'talk' : "Talks"
        };
    </script>
    

    
    

    
    
    <script id="search-hit-fuse-template" type="text/x-template">
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js" integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script>
    

    
    

    
    
    
    
    
    
    
    
    
      
    
    
    
    
    <script src="/js/academic.min.ac3764a0b2acc1c82ace08e756adafc1.js"></script>

    






  
  <div class="container">
    <footer class="site-footer">
  
  <p class="powered-by">
    <a href="/privacy/">Privacy Policy</a>
  </p>
  

  <p class="powered-by">
    &copy; L. Dávalos 2019 &middot; 

    Powered by the
    <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic theme</a> for
    <a href="https://gohugo.io" target="_blank" rel="noopener">Hugo</a>.

    
    <span class="float-right" aria-hidden="true">
      <a href="#" id="back_to_top">
        <span class="button_icon">
          <i class="fas fa-chevron-up fa-2x"></i>
        </span>
      </a>
    </span>
    
  </p>
</footer>

  </div>
  

  
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

</body>
</html>
